principles.md

```diff
@@ -0,0 +1,34 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web, also known as web3. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+**Liberty**
+We believe in the sovereignty of individuals. As a platform that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms.
+
+**Censorship resistance**
+
+We aim for a free flow of information. No content is under surveillance. One of the design goals of Ethereum is to be censorship resistant, and we should provide the same guarantees across Status.
+
+**Security**
+We don't compromise on security when building features. We will use state-of-the-art technologies, and research new security methods and technologies to make strong security guarantees.
+
+**Privacy**
+Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as to preserve the right to total anonymity.
+
+**Transparency**
+We strive for complete openness and symmetry of information, and have no border between our core contributors and our community. We will be frank about our shortcomings, especially when making short-term tradeoffs in service of our long-term goals.
+
+**Free culture**
+The software we create is a public good. It will be made available via a free and open source license, for anyone to share, modify and benefit from.
```

Comment:
maybe just: `...it _is_ made via a free and open source...` because it is open-source already

Reply:
Good point, agree

Reply:
@MaxRis how would you suggest we do this? To me it seems to be included in the "free culture" (i.e. to me MIT/BSD are more free than GPL, e.g.)

Reply:
@oskarth I've tried to investigate if already exists some license, as a sample, which allows re-licensing, but was not able to find such so far.
Recently, someone created following issue https://github.com/status-im/react-native-desktop/issues/304 . It seems in some cases MIT license is preferable than BSD+PATENTS license.
Wondering how good might be idea to create Status own license which explicitly will allow source code re-licensing to the any license from the allowed licenses list (the list can be extended in the future, but currently can contain MIT, BSD, etc.).
At the other hand  https://opensource.org/licenses suggests to not create new licenses duplicating already available licenses.

Reply:
One thing the Rust community does for licensing versatility is they dual license MIT + Apache2.0. Apache2.0 provides a lot of friendlier patent grants than the BSD+Patents combo, and MIT is provided because Apache2 is incompatible with GPLv2. Here's an issue on this matter on a Rust project:

https://github.com/sfackler/rust-postgres-macros/issues/19

Reply:
Thank you @empyrical , this case helps a lot! Sounds like we can define and use any range of desired licenses at the same time.

Reply:
we're using MIT + Apachev2 for Nimbus for these precise same reasons, (also looked to rust for inspiration) - it seemed like the way forwards to offer the broadest reach from a license-mixing point of view

Reply:
ie we allow the user to choose between apachev2 and MIT as they like - best of both worlds and no need to invent a new license